custom_components/waste_collection_schedule/waste_collection_schedule/source/cincinnati_gov.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime, timedelta
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "City of Cincinnati, OH"
+DESCRIPTION = "City of Cincinnati, OH, USA"
+URL = "https://www.cincinnati.gov/"
+COUNTRY = "us"
+TEST_CASES = {}
+HEADERS = {"user-agent": "Mozilla/5.0"}
+ICON_MAP = {
+    "Trash": "mdi:trash-can",
+    "Recycling": "mdi:recycle",
+    "Street Sweeping": "mdi:road-variant",
+}
+
+# ### Arguments affecting the configuration GUI ####
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = {  # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "Find your address ID by searching for your address on Cincinnati's service website. The address ID is needed to retrieve collection schedules.",
```

Comment:
maybe explain that you can find the id in the URL after searching for the address